test/checkpoint/workers.c

```diff
@@ -239,56 +239,52 @@ real_worker(void)
             (void)log_print_err("session.open_cursor", ret, 1);
             goto err;
         }
-    has_cursors = true;
 
-    if (g.use_timestamps)
-        begin_cfg = "read_timestamp=1,roundup_timestamps=(read=true)";
-    else
-        begin_cfg = NULL;
+    if ((ret = session->begin_transaction(session, NULL)) != 0) {
+        (void)log_print_err("real_worker:begin_transaction", ret, 1);
+        goto err;
+    }
 
     for (i = 0; i < g.nops && g.running; ++i, __wt_yield()) {
-        if ((ret = session->begin_transaction(session, begin_cfg)) != 0) {
-            (void)log_print_err("real_worker:begin_transaction", ret, 1);
-            goto err;
-        }
         keyno = __wt_random(&rnd) % g.nkeys + 1;
-        if (g.use_timestamps && i % 23 == 0) {
```

Comment:
Is this a change of behaviour?

Reply:
In theory yes but its handling the difference in how frequently we commit the transaction due to the difference in taking the lock / writing the commit timestamp. I think this behaviour change is fine as we still do a similar number of operations regardless.